packages/extension/src/newtab/DndModal.tsx

```diff
@@ -105,23 +114,22 @@ const DoNotDisturbModal: React.FC<DoNotDisturbModalProps> = ({
       }}
     >
       <ModalCloseButton onClick={onRequestClose} />
-      <div className={styles.heading}>
-        <h3 className={styles.title}>Do Not Disturb</h3>
-        {isActive ? null : (
-          <p className={styles.description}>
-            Choose your preferences while you&apos;re on Do Not Disturb mode
-          </p>
-        )}
-      </div>
-      {renderForm()}
-      <div className={classnames(styles.footer, styles.centered)}>
-        <button
-          className={classnames(styles.done, styles.centered)}
-          onClick={(e) => (isActive ? setDndSettings(null) : handleSubmit(e))}
-        >
-          {isActive ? 'Turn off' : 'Done'}
-        </button>
+      <div className="px-16 text-center">
+        <h3 className="font-bold text-2xl text-white">Do Not Disturb</h3>
+        <p className="mt-1 text-sm text-salt-50">{getDescription()}</p>
       </div>
+      {isActive ? turnedOnContent : turnedOffContent}
+      {isActive ? null : (
+        <div className="flex flex-row justify-center items-center w-full py-5 border-t border-salt-90 border-opacity-40">
+          <button
```

Comment:
The same comment about the buttons applies here. This time the relevant class is not btn-secondary but `btn-primary`.